include/sisl/flip/flip.hpp

```diff
@@ -438,10 +438,13 @@ class Flip {
     }
 
     void start_rpc_server() {
-        m_flip_server_thread = std::unique_ptr< std::thread >(new std::thread(FlipRPCServer::rpc_thread));
+        m_flip_server_thread =
+            std::unique_ptr< std::thread >(new std::thread([this]() { m_flip_server.rpc_thread(); }));
         m_flip_server_thread->detach();
     }
 
+    void stop_rpc_server() { m_flip_server.shutdown(); }
```

Comment:
I would suggest adding m_flip_server.reset() as well here, otherwise flipserver destructs only singleton is out of scope.

Reply:
Ack